src/account/UpgradeableModularAccount.sol

```diff
@@ -606,4 +609,42 @@ contract UpgradeableModularAccount is
 
     // solhint-disable-next-line no-empty-blocks
     function _authorizeUpgrade(address newImplementation) internal override {}
+
+    function _toFunctionReference(bytes32 setValue) internal pure returns (FunctionReference) {
+        return FunctionReference.wrap(bytes21(setValue));
+    }
+
+    function _installPlugin(
+        address plugin,
+        bytes32 manifestHash,
+        bytes memory pluginInitData,
+        FunctionReference[] memory dependencies,
+        InjectedHook[] memory injectedHooks
+    ) internal {
+        bytes memory data = abi.encodeCall(
+            PluginManager.installPlugin, (plugin, manifestHash, pluginInitData, dependencies, injectedHooks)
+        );
+        (bool success, bytes memory returndata) = address(_PLUGIN_MANAGER).delegatecall(data);
+        if (!success) {
+            assembly ("memory-safe") {
+                revert(add(returndata, 32), mload(returndata))
+            }
+        }
```

Comment:
Nit: 
Not necessary to extract the logic here and below in `_uninstallPlugin` into internal functions as logic is simple and not reused.

Reply:
`_installPlugin` is reused by `initialize` and `installPlugin`, so I'd prefer to keep it as a function. I can move the contents of `_uninstallPlugin` though.